src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,53 @@
-export const Tabs = () => {};
+export type Tab = {
+  id: string
+  title: string
+  content: string
+};
+
+type TabsProps = {
+  tabs: Tab[]
+  selectedTabId: string
+  onTabSelected: (tab: Tab) => void
+};
+
+export const Tabs = ({ tabs, selectedTabId, onTabSelected }: TabsProps) => {
+  let selectedTabIdCopy = selectedTabId;
+
+  if (tabs.length > 0 && !tabs.map(tab => tab.id).includes(selectedTabId)) {
```

Comment:
think this if is redundant here, better extract this logic to separate functions

Reply:
I deleted this code.